filesystem/README.md

```diff
@@ -9,6 +9,15 @@ npm install @capacitor/filesystem
 npx cap sync
 ```
 
+## iOS
+
+To have files appear in the Files app, you must set the following keys to `YES` in `Info.plist`:
+
+- `UIFileSharingEnabled` (`Application supports iTunes file sharing`)
+- `LSSupportsOpeningDocumentsInPlace` (`Supports opening documents in place`)
+
+Read about [Configuring `Info.plist`](https://capacitorjs.com/docs/ios/configuration#configuring-infoplist) in the [iOS Guide](https://capacitorjs.com/docs/ios) for more information on setting iOS permissions in Xcode.
```

Comment:
I guess this code is copied from other plugins such as camera or geolocation, but not sure where it comes from. I don't think the whole `in the [iOS Guide](https://capacitorjs.com/docs/ios) for more information on setting iOS permissions in Xcode` part is relevant anymore for those plugins as that section doesn't include any information related to the Info.plist
And in particular for this change is less relevant since this is not a permission.

Can be changed to just 
```suggestion
Read about [Configuring `Info.plist`](https://capacitorjs.com/docs/ios/configuration#configuring-infoplist) for help.
```
As [status bar plugin](https://github.com/ionic-team/capacitor-plugins/tree/main/status-bar#ios-note) does

Reply:
Thanks, I found it in the `README.md` file for the Camera plugin. I've committed the change.